### PR TITLE
[TN-3]Add gitignore readme template

### PR DESCRIPTION
**Pull request by AdrianDVL**
Adding files:
.gitignore
README
Template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## This is a github trainning repo
```
Comment (enriqu3l) [inline README.md]: Typo. Should be **training**

**Review by enriqu3l — COMMENTED** [review]
I found a typo.

File: README.md (hunk 1 above)
Comment (AdrianDVL) [reply README.md]: Thanks @enriqu3l I already fixed

**Review by enriqu3l — APPROVED** [review]
LGTM

**Pull request merged**